.cirrus.yml

```diff
@@ -32,7 +32,7 @@ env:
     DEBIAN_NAME: "debian-13"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c20230614t132754z-f38f37d13"
+    IMAGE_SUFFIX: "c20230426t140447z-f38f37d12"
```

Comment:
Uh oh.  Those are date/time stamps.  I think you're going backwards here :smile: 

I'm guessing you want https://github.com/containers/podman/pull/18612 which (currently) depends on https://github.com/containers/automation_images/pull/284 and possibly more builds after that depending on what breaks next :cry: